src/core_ocean/shared/mpas_ocn_tracer_hmix_redi.F

```diff
@@ -189,7 +189,7 @@ subroutine ocn_tracer_hmix_Redi_tend(meshPool, layerThickness, layerThicknessEdg
       !$omp end do
       !$omp end parallel
 
-      nCells = nCellsArray(1)
```

Comment:
@vanroekel the Redi hmix was only computing on the owned cells here, but involves T&S on neighboring cells. So the ice sheet melt fluxes must alter T&S on halo 1 and caused the PEM fail, while non-ISMF did not alter. We almost never compute *only* on owned cells, so extending the computation to halo 1 here seems quite reasonable.

Reply:
Thanks, that makes sense.  I agree extending one is reasonable.  I was just curious why this fix did it.